allennlp/training/trainer.py

```diff
@@ -122,22 +122,29 @@ def __call__(
         epoch: int,
         batch_number: int,
         is_training: bool,
+        is_master: bool,
```

Comment:
You don't need this, do you?  It's already queryable from the `trainer` argument.  You also didn't add it to the `EpochCallback`.